spec/requests/providers_spec.rb

```diff
@@ -1791,6 +1791,52 @@ class DummyProvider; end
     end
   end
 
+  context 'configuration_profiles subcollection' do
+    let(:configuration_profile) { FactoryBot.create(:configuration_profile) }
+    let(:ems) { FactoryBot.create(:configuration_manager) }
+
+    context 'GET /api/providers/:id/configuration_profiles' do
+      it 'returns the lans with an appropriate role' do
```

Comment:
I just realized, the it comments here and below mention lans instead of configuration profiles.

Reply:
Copy/past fails again, fixed